src/main/java/io/split/android/client/cache/SplitCache.java

```diff
@@ -38,15 +41,15 @@
 
     private long mChangeNumber = -1;
     private Set<String> mRemovedSplits = null;
-    private Map<String, Split> mInMemorySplits = null;
-    private Map<String, Integer> mTrafficTypes = null;
+    private ConcurrentHashMap<String, Split> mInMemorySplits = null;
```

Comment:
consider using Map and concurrentHashMap in the right side.

Reply:
My bad. Fixed. Thanks!